includes/definitions/advafsp150eg-x.yaml

```diff
@@ -0,0 +1,13 @@
+os: advafsp150eg-x
```

Comment:
This os name is too specific, especially seeing as you say it covers another device.

The OS should be named after the operating system / firmware which is run on these devices. If they don't have one then it should be the group of products this covers.